### DISCUSSION TITLE
Voice Web on Reticulum

### DISCUSSION THREAD

**Discussion by ProgrammerWhoPrograms** [markqvist/Reticulum#339] [Ideas]
The [W3C Voice Browser Working group](https://www.w3.org/Voice/) has unfortunately been discontinued. Which is a shame, as I think would be very useful for blind people.
Anyway, one of the apparent problems with it in my opinion is that it works via insecure PSTN and is based on 'server-side' (for want of a better word) processing so you can't download a page to listen to later.
Having it run over Reticulum instead of the phone network should solve those problems.
Also, if it exists, it will increase the attention Reticulum gets, which should increase the size of the network, the number of developers interested, and the amount of money donated (which should help pay for an audit).
What do you think?

**Comment by markqvist** [comment]
This is interesting, and something I wish I had time to dive into much more deeply, but unfortunately I don't have that right now. Maybe someone else will pick up the ball.
